src/pages/ReportDetailsPage.tsx

```diff
@@ -395,7 +395,7 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
             });
         }
 
-        if (policy && connectedIntegration && isPolicyAdmin && !isSingleTransactionView) {
+        if (policy && connectedIntegration && isPolicyAdmin && !isSingleTransactionView && !isThread) {
```

Comment:
I think we need to use something else, otherwise it won't show in the parent report either...

I think `isExpenseReport` works:
<img width="1023" alt="Screenshot 2024-07-24 at 16 55 33" src="https://github.com/user-attachments/assets/6d83380f-3b29-401d-a441-b55729c18120">

As opposed to `!isThread`:
<img width="1023" alt="image" src="https://github.com/user-attachments/assets/5b57562a-f206-422f-bbc8-203e19cb8532">

Reply:
My reasoning was that there's already a big button in the report header but yeah your suggestion makes sense.

<img width="1130" alt="image" src="https://github.com/user-attachments/assets/f7834164-b15e-4e4b-8ba8-8fe5082f4b09">